docs/jquery.md

```diff
@@ -30,27 +30,54 @@ gem 'opal-jquery'
 
 #### Usage
 
-`opal-jquery` can now be easily added to your opal application sources using a
-standard require:
+`opal-jquery` can now be easily added to your opal application sources:
 
 ```ruby
 # app/application.rb
-require 'opal'
-require 'jquery'
-require 'opal-jquery'
+require 'opal/jquery'
```

Comment:
~~requiring opal and jquery separately is still needed, any reason for this change and the switch to `/`?~~

I see, it's stated below, can you add a comment or indication on the lines of "see below for a full example"?

Reply:
Comment added! When I first made this pull request, using `require 'opal/jquery'` was the only way I could make this example work. Now that I understand Opal better, I see that `require 'opal-jquery'` can work if the user compiles their app with the added flag: `--gem=opal-jquery`.

I personally prefer `require 'opal/jquery'` because it always works, no matter how you build your dependencies.